utils/src/main/java/org/corfudb/utils/lock/persistence/LockStore.java

```diff
@@ -51,7 +51,7 @@
     /**
      * Cache of all the observed locks/leases. Contains the last timestamp at which the lock was last observed.
      */
-    private Map<LockId, ObservedLock> observedLocks = new ConcurrentHashMap<>();
+    private final Map<LockId, ObservedLock> observedLocks = new ConcurrentHashMap<>();
```

Comment:
would be great to use `ConcurrentMap` which tells the user that the map is concurrent

Reply:
agree, done